app/controllers/concerns/verify_setup_completed.rb

```diff
@@ -41,4 +41,8 @@ def verify_setup_completed
   def config_path?
     request.env["PATH_INFO"] == admin_config_path
   end
+
+  def creator_setup_path?
+    request.env["PATH_INFO"] == "/admin/creator_settings/new"
+  end
```

Comment:
I've attempted to use the path helper here (`new_admin_creator_setting_path`), but it doesn't appear to work. This, however, works as intended. 🤔

Reply:
@juliannatetreault with the current solution, I am still getting very inconsistent behaviour. See gif below. Sometimes it works on a refresh whilst at other times it does not. 
![banner](https://user-images.githubusercontent.com/2786819/141456489-cd6eed72-9988-4afa-b509-0c710c65402c.gif)

It seems that there is something more going on with the path helpers here because as you hinted that `new_admin_creator_setting_path` and `/admin/creator_settings/new` are the same thing. 

Maybe try investigating what the outcome of ` request.env["PATH_INFO"]` is when you get to this page vs when you get to other pages. There may be something weird happening there.

Reply:
I went down a rabbit hole debugging this on multiple browsers (chrome _and_ safari) and have been able to get `request.env["PATH_INFO"] == new_admin_creator_setting_path` to work consistently across both browsers. Additionally, the setup banner shows all designated pages other than the Creator Settings form page. Let me know if you still experience unexpected behavior, @Ridhwana!